src/components/dashboard/main-content/course-enrollments/tests/CourseEnrollments.test.jsx

```diff
@@ -226,6 +231,46 @@ describe('Course enrollments', () => {
     expect(mockAcknowledgeAssignments).toHaveBeenCalledWith({ assignmentState: ASSIGNMENT_TYPES.EXPIRED });
   });
 
+  it(
```

Comment:
We should maybe add in a test if they've been added to multiple groups at once

Reply:
I added the test in the hook for when a user is added to multiple groups!